test/utils/copy-db-date.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var db             = require('../../db')
+  , defineDbjsDate = require('dbjs-ext/date-time/date');
+
+module.exports = function (t, a) {
+	var date, dbDate;
+	defineDbjsDate(db);
+	date   = new db.Date();
+	dbDate = t(date);
+	date.setUTCFullYear(date.getUTCFullYear());
+	date.setUTCMonth(date.getUTCMonth());
+	date.setUTCDate(date.getUTCDate());
```

Comment:
What's the intention behind code in above three lines? It seems redundant, having zero effect